ctapipe/io/tests/test_simteleventsource.py

```diff
@@ -517,6 +524,46 @@ def test_simtel_no_metadata(monkeypatch):
         assert all([t.optics.name.startswith("UNKNOWN") for t in subarray.tel.values()])
 
 
+def test_load_atmosphere_from_simtel(prod5_gamma_simtel_path):
+    """
+    Load atmosphere from a SimTelEventSource
+    """
+    from ctapipe.atmosphere import (
+        FiveLayerAtmosphereDensityProfile,
+        TableAtmosphereDensityProfile,
+    )
+
+    # old simtel files don't have the profile in them, so a null list should be returned
```

Comment:
This comment is wrong here? Below is correct.

Reply:
Yup, cut-and-paste error I think! Fixed.